packages/pds/src/api/com/atproto/admin/sendEmail.ts

```diff
@@ -0,0 +1,46 @@
+import { AuthRequiredError, InvalidRequestError } from '@atproto/xrpc-server'
+import { Server } from '../../../../lexicon'
+import AppContext from '../../../../context'
+
+export default function (server: Server, ctx: AppContext) {
+  server.com.atproto.admin.sendEmail({
+    auth: ctx.roleVerifier,
+    handler: async ({ input, auth }) => {
+      if (!auth.credentials.admin && !auth.credentials.moderator) {
+        throw new AuthRequiredError('Insufficient privileges')
+      }
+
+      try {
+        const {
+          content,
+          recipientDid,
+          subject = 'Message from Bluesky moderator',
+        } = input.body
+        const userInfo = await ctx.db.db
+          .selectFrom('user_account')
+          .where('did', '=', recipientDid)
+          .select('email')
+          .executeTakeFirst()
+
+        if (!userInfo) {
+          throw new InvalidRequestError('Recipient not found')
+        }
+
+        await ctx.moderationMailer.send(
+          { content },
+          { subject, to: userInfo.email },
+        )
+        return {
+          encoding: 'application/json',
+          body: { sent: true },
+        }
+      } catch (err) {
+        return {
+          encoding: 'application/json',
+          // TODO: add more details about the error via message?
+          body: { sent: false },
+        }
```

Comment:
Is there a special reason why we want to use a 200/success response here rather than an error response?  For example, the error thrown above `throw new InvalidRequestError('Recipient not found')` will end-up being turned into a 200 rather than remaining 400 with some error details.

Reply:
ahh good catch! I originally wrapped it this way to capture any potential error from the actual send email request and return it to the client with the `message` property but you're right, 200 status code wouldn't be right in that case either. I'll get rid of `message` prop from the response as well.